client/src/main/java/org/eclipse/hono/client/CommandResponse.java

```diff
@@ -35,7 +35,8 @@
     private final String replyToId;
     private final String correlationId;
 
-    private CommandResponse(final Buffer payload, final String contentType, final int status, final String correlationId, final String replyToId) {
+    private CommandResponse(final String tenantId, final String deviceId, final Buffer payload,
```

Comment:
this doesn't look right, neither *tenantId* nor *deviceId* are used ...

Reply:
@sophokles73 These _tenantId_ and _deviceId_ are used in the next part of the PR to set those values in amqp message.

Reply:
then please remove the unused parameters

Reply:
If I remove them here, then it will be still unused in `CommandResponse.from(..` and it has a chain effect.Then device and tenant ids cannot be brought into the `CommandResponse`. I will add one next step (setting inside properties), so that these ids are used.